build/vega-lite-schema.json

```diff
@@ -5554,6 +5554,14 @@
           "$ref": "#/definitions/Orientation",
           "description": "The default direction (`\"horizontal\"` or `\"vertical\"`) for gradient legends.\n\n__Default value:__ `\"vertical\"`."
         },
+        "gradientHorizontalMaxLength": {
+          "description": "Max legend length for a horizontal gradient.\n\n__Default value:__ `200`",
```

Comment:
This is a bit misleading since the user can set a gradient length that is outside these bounds, no?